pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -303,6 +303,8 @@ func ReadLogs(ctx context.Context, path, containerID string, opts *LogOptions, r
 		return fmt.Errorf("failed to seek %d in log file %q: %v", start, path, err)
 	}
 
+	limitedMode := (opts.tail >= 0) && (!opts.follow)
```

Comment:
just noting:

the default is -1 for tail (read all log) and default for follow is false.